specification/databricks/resource-manager/Microsoft.Databricks/preview/2024-09-01-preview/databricks.json

```diff
@@ -1153,7 +1153,8 @@
       "enum": [
         "NONE",
         "HIPAA",
-        "PCI_DSS"
+        "PCI_DSS",
+        "CYBER_ESSENTIAL_PLUS"
```

Comment:
@curiekim00 Is there a precedence for including location restrictions in Swagger specs? We should at least check if this exists.

Reply:
Hi @scottmacchia-msft I have checked that there was no precedence for including location restrictions and have double checked with the Breaking Change reviewers that it is not possible to add validations in the spec, and suggested just documenting this restriction